src/views/RematchView.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div id="game-view-wrapper">
+    <!-- Unauthenticated/Must re-log in/ Unavailable game -->
+    <template v-if="$store.state.game.myPNum === null">
+      <game-unavailable-view />
+    </template>
+
+    <!-- Authenticated View -->
+    <template v-else>
+      <div id="game-menu-wrapper" class="d-flex flex-column flex-sm-row align-center">
+        <game-menu :isSpectating="false" />
+      </div>
+
+      <BaseSnackbar
+        v-model="showSnackbar"
+        :message="snackBarMessage"
+        :color="snackColor"
+        data-cy="game-snackbar"
+        @clear="clearSnackBar"
+      />
+      <div class="game-overlays">
+        <v-overlay id="waiting-for-game-to-start-scrim" v-model="waitingForGameToStart" class="game-overlay">
+          <h1 :class="[this.$vuetify.display.xs === true ? 'text-h5' : 'text-h3', 'overlay-header']">
+            Waiting for Game to Start
+          </h1>
+          <v-btn
+            color="secondary"
+            class="mt-4"
+            data-cy="leave-unstarted-game-button"
+            :loading="leavingGame"
+            @click="goHome"
+          >
+            Leave Game
+          </v-btn>
+        </v-overlay>
+      </div>
+    </template>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapState } from 'vuex';
+
+import { ROUTE_NAME_HOME } from '@/router';
+import GameMenu from '@/components/GameView/GameMenu.vue';
+import GameOverlays from '@/components/GameView/GameOverlays.vue';
+import GameUnavailableView from '../components/GameView/GameUnavailableView.vue';
+import UsernameToolTip from '@/components/GameView/UsernameToolTip.vue';
+import BaseSnackbar from '@/components/Global/BaseSnackbar.vue';
+
+export default {
+  name: 'RematchView',
+  components: {
+    GameMenu,
+    GameOverlays,
+    GameUnavailableView,
+    UsernameToolTip,
+    BaseSnackbar,
+  },
+  data() {
+    return {
+      leavingGame: false,
+      showSnackbar: false,
+      snackBarMessage: '',
+      snackColor: 'error',
```

Comment:
Since this never changes the snack color, it could hard code the `error` color into the template instead of using a reactive variable
```suggestion
```